packages/tpetra/core/src/Tpetra_Details_DefaultTypes.hpp

```diff
@@ -73,20 +73,39 @@ namespace DefaultTypes {
 
   /// \typedef global_ordinal_type
   /// \brief Default value of GlobalOrdinal template parameter.
-#if defined(HAVE_TPETRA_INST_INT_INT)
-  typedef int global_ordinal_type;
-#elif defined(HAVE_TPETRA_INST_INT_LONG_LONG)
-  typedef long long global_ordinal_type;
-#elif defined(HAVE_TPETRA_INST_INT_LONG)
-  typedef long global_ordinal_type;
-#elif defined(HAVE_TPETRA_INST_INT_UNSIGNED_LONG)
-  typedef unsigned long global_ordinal_type;
-#elif defined(HAVE_TPETRA_INST_INT_UNSIGNED)
-  typedef unsigned global_ordinal_type;
-#else
-#  error "Tpetra: No global ordinal types in the set {int, long long, long, unsigned long, unsigned} have been enabled."
+#if defined(TPETRA_ENABLE_DEPRECATED_CODE)
+    // NEW style picks `int` for Global Ordinal as the first choice.
```

Comment:
Not sure what "NEW style" means -- wouldn't this be old style?  Anyway, the comment is redundant (just reading the code should explain which type is picked first), but harmless.

Reply:
meh, sure.  what I get working on my day off in a sleep deprived state :p

Reply:
we can shoot the comment in the head now or wait till after the major release happens and we nuke all the deprecated stuff.

Reply:
@william76 It's totally cool -- we can fix it later (since we'll be touching all these soon again anyway).